suseconnect/usage.txt

```diff
@@ -40,6 +40,9 @@ Manage subscriptions at https://scc.suse.com
         --write-config       Write options to config file at /etc/SUSEConnect.
         --cleanup            Remove old system credentials and all zypper
                              services installed by SUSEConnect.
+        --rollback           Revert the registration state in case of a failed
+                             migration.
+
 
```

Comment:
The Ruby version has just one blank line before `Common options:`

Reply:
good catch! fixed.